core/components/molecules/pager/pager.js

```diff
@@ -1,47 +1,69 @@
-import React from 'react'
-import PropTypes from 'prop-types'
-import styled from 'styled-components'
-import Button from '../../atoms/button'
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
 
-import { changePageIfAppropiate, totals } from '../../_helpers/pagination'
+import Button from "../../atoms/button";
+import Icon from "../../atoms/icon";
+import { changePageIfAppropiate, totals } from "../../_helpers/pagination";
 
 const StyledPager = styled.div`
   display: flex;
   justify-content: space-between;
   align-items: center;
-`
+`;
```

Comment:
😱 `;;;;;`

Some one needs to review their `precommit` hook :)

Reply:
Ohhh. I don't know what happened there. Checking...